plugins/metrics/commands/ai-docs-telemetry.md

```
@@ -0,0 +1,97 @@
+---
+description: Analyze Claude Code session logs for ai-docs usage patterns
+argument-hint: "[-scan] [-project <name>] [-session <path>]"
+---
+
+## Name
+metrics:ai-docs-telemetry
+
+## Synopsis
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)